.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
+    "python.testing.pytestArgs": [
+        "src"
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true,
+    "python.analysis.extraPaths": [
+        "./src"
+    ]
+}
```

Comment:
Is the testing required to work in vscode? Could this settings be generated automatically by vscode?

Reply:
This is for the VSCode Extention to locate the tests. The tests themselves can work independently. Currently there are some issues with the import so some minor modifications are required before conducting a test.